### DISCUSSION TITLE
PaddleServing服务部署时，如果需要使用表格识别模型，怎么修改config.yml？

### DISCUSSION THREAD

**Discussion by asenasen123** [PaddlePaddle/PaddleOCR#14205] [Q&A]
PaddleServing服务部署时，如果需要使用表格识别模型，怎么修改config.yml？
https://github.com/PaddlePaddle/PaddleOCR/blob/main/deploy/pdserving/config.yml

**Comment by UserWangZz** [comment]
可以参考paddlex的方式进行部署
